rollbar-java/src/main/java/com/rollbar/notifier/sender/queue/DiskQueue.java

```diff
@@ -43,12 +43,12 @@ private DiskQueue(Builder builder) {
 
     if (!this.queueFolder.exists()) {
       if (!this.queueFolder.mkdirs()) {
-        throw new RuntimeException("Could not create folder: " + queueFolder);
```

Comment:
What are the implications of the app running without this folder?

Reply:
Errors will not be saved for the session, but since it is not possible to create a folder in which to save the information, it is not appropriate to generate a crash for a user.

Reply:
These are local crash reports/logs?

Reply:
this are payloads that we save to send them later